.gitignore

```diff
@@ -1,3 +1,6 @@
+# untracked data
```

Comment:
Small point, but could make the comment more descriptive - e.g. `# CSV and data files pulled from a remote source`